src/components/BarList/BarLabel.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC, ReactNode } from 'react'
+import { composeClasses } from 'lib/classes'
+import { Size } from 'interfaces'
+
+interface IPropsBarLabel {
```

Comment:
we don't need add suffix on interfaces

```suggestion
export interface BarLabelProps {
```